Description: Remove DTDs from examples
 By default, dia installs dtds duplicated in several language's example/
 documentation directories. This is not necessary for Debian (installing
 DTDs in standard catalog registration locations). Therefore removing.
Author: Roland Stigge <stigge@antcom.de>

--- dia-0.97.2.orig/doc/eu/Makefile.am
+++ dia-0.97.2/doc/eu/Makefile.am
@@ -72,11 +72,6 @@ metadata = \
 	dia-eu.omf \
 	topic.dat
 
-examples = \
-	../shape.dtd \
-	../diagram.dtd \
-	../sheet.dtd
-
 ##	$(progname).$(progfiles_extension)
 
 DISTCLEANFILES = 
--- dia-0.97.2.orig/doc/en/Makefile.am
+++ dia-0.97.2/doc/en/Makefile.am
@@ -153,11 +153,6 @@ metadata = \
 	dia-C.omf \
 	topic.dat
 
-examples = \
-	../shape.dtd \
-	../diagram.dtd \
-	../sheet.dtd
-
 
 DISTCLEANFILES = 
 
--- dia-0.97.2.orig/doc/pl/Makefile.am
+++ dia-0.97.2/doc/pl/Makefile.am
@@ -73,10 +73,6 @@ metadata = \
 	dia-pl.omf \
 	topic.dat
 
-examples = \
-	../shape.dtd \
-	../diagram.dtd \
-	../sheet.dtd
 
 ##	$(progname).$(progfiles_extension) \
 
--- dia-0.97.2.orig/doc/fr/Makefile.am
+++ dia-0.97.2/doc/fr/Makefile.am
@@ -76,10 +76,6 @@ metadata = \
 	dia-fr.omf \
 	topic.dat
 
-examples = \
-	../shape.dtd \
-	../diagram.dtd \
-	../sheet.dtd
 
 DISTCLEANFILES = 
 
